src/ProgressIndicator/ProgressIndicator.svelte

```diff
@@ -16,8 +16,8 @@
 
   const dispatch = createEventDispatcher();
   const steps = writable([]);
-  const stepsById = derived(steps, ($) =>
-    $.reduce((a, c) => ({ ...a, [c.id]: c }), {})
+  const stepsById = derived(steps, (stepStore) =>
+    stepStore.reduce((a, c) => ({ ...a, [c.id]: c }), {})
```

Comment:
`steps` is the store, this is the content.

Only `$` on its own is reserved, so in cases like this, I would go with the conventional `$steps` which is analogous to the name in auto-subscriptions.